packages/atomic/src/components/common/checkbox.ts

```diff
@@ -0,0 +1,72 @@
+import {html, TemplateResult} from 'lit';
+import {classMap} from 'lit/directives/class-map.js';
+import {ifDefined} from 'lit/directives/if-defined.js';
+import {ref, RefOrCallback} from 'lit/directives/ref.js';
+import Tick from '../../images/checkbox.svg';
+
+export interface CheckboxProps {
+  checked: boolean;
+  onToggle(checked: boolean): void;
+  key?: string | number;
+  id?: string;
+  class?: string;
+  text?: string;
```

Comment:
should this really be optional?

Reply:
There are currently component calling the Checkbox with no text value (e.g. FacetValueCheckbox).
I think it makes sense to keep it optional